tools/performance-server/src/main/kotlin/routes/route.kt

```diff
@@ -64,6 +64,22 @@ object LocalCache {
     fun buildExists(target: String, buildNumber: String) =
             buildsInfo[target][buildNumber]?.let { true } ?: false
 
+    fun delete(target: String, builds: Iterable<String>, bintrayUser: String, bintrayPassword: String) {
+        // Delete from bintray.
+        val buildsDescription = getBuildsInfoFromBintray(target).lines()
+        buildsDescription.filter {
+            val buildNumber = it.substringBefore(',')
+            buildNumber !in builds
+        }
+        // Upload new version of file.
+        val uploadUrl = "$uploadBintrayUrl/$bintrayPackage/latest/$target/$buildsFileName?publish=1&override=1"
+        sendUploadRequest(uploadUrl, buildsDescription.joinToString("\n"), bintrayUser, bintrayPassword)
```

Comment:
Why it uploads smth?

Reply:
It removes data from file with description of builds and should renew it on bintray.